includes/diagnostics/class-wc-stripe-diagnostics-redactor.php

```
@@ -0,0 +1,327 @@
+			'~(https?://[^\s?]+)\?[^\s]*~',
+			static function ( $m ) {
+				return $m[1] . '?[redacted]';
+			},
+			$value
+		);
+		if ( is_string( $replaced ) ) {
+			$value = $replaced;
+		}
+
+		return $value;
+	}
```

PHPMD (2.15.0)

[error] The method scrub_string is not named in camelCase. (undefined)

(CamelCaseMethodName)